### PR TITLE
# Need Review for Submit Nomination code

### PR DESCRIPTION
**Pull request by rengaswa-dv**
Hi,
I have added the Submit_nomination Draft Code. Please review and share your comments

### CODE REVIEW

File: Submit_nomination [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+promoName = data("Promotion_Name")'"GM-Nomination"
+nomineeType = data("Team")'"Individual"
+nomineeBehavior = "Ideas"
+nomineeCard = "1st Year Celebration"
+nomineeComment = "Bob is the hardest working person that I know. he is the first to arrive and the last to leave." & _
+				 "he is always positive and provides a pleasant working environment. He always greets you with a smile. Bob is the rock of the QA services" & _
+				 "he is what makes us great! Bob’s outstanding performance is a daily occurrence. Examples of his outstanding performance can be witnessed" &_
+				 "and appreciated whenever you have an interaction with him. Everyday he provides a can do attitude that is contagious to everybody."
+
+searchBy = "Last Name"
```
Comment (mohamedismail86) [inline Submit_nomination]: Here my Comments are:
1. searchBy field should be dynamic and add more variables
2. Remove Wait functions
Defect has been raised the same. Once you fix the code make the update in issue

**Comment by rengaswa-dv** [comment]
Issue #8 has been close. please review

**Comment by rengaswa-dv** [comment]
Issue #8 closed.

**Comment by mohamedismail86** [comment]
Looks fine now. You can Merge

**Comment by rengaswa-dv** [comment]
Pull request successfully merged and closed. Hence Deleting the Branch

**Pull request merged**